### PR TITLE
Adds A Third Non-Conductive Tier

### PR DESCRIPTION
**Pull request by EtheoBoxxman**
Wasnt present in my last pr to enable custom species besks, because I didnt know what made creatures resistant to electric shocks.
This just adds a new expensive trait that makes you immune to electric shocks. Costs 6 points, and its basically just in-built insuls.

## Changelog

:cl:
add: Changes Non-Conductive and Non-Conductive, Major to 'Non-Conductive, Minor' and 'Non-Conductive'.
add: Adds Non-Conductive, Major. Costing 6 points.
/:cl:

### CODE REVIEW

**Comment by Skysteak** [comment]
Just to double check, does this effect tazers or stun batons?

**Comment by EtheoBoxxman** [comment]
> To double check, does this effect tazers or stun batons? I know you said 'like having insuls' but just in case people get paranoid.

It affects tasers, according to comments on the other traits. O:

**Comment by Aroliacue** [comment]
I think 6 points is a little much but overall this is cool

File: code/__defines/species_languages_vr.dm [hunk 1]
```diff
@@ -18,8 +18,9 @@
 #define LANGUAGE_MOUSE "Mouse"
 
 #define LANGUAGE_SHADEKIN "Shadekin Empathy"
+#define LANGUAGE_MARISH "Marish" // CHOMPedit
```
Comment (Kashargul) [inline code/__defines/species_languages_vr.dm:21]: Why is it not directly under the chomp languages? There's an entire CHOMPEdit language section below

File: code/modules/mob/language/station_vr.dm [hunk 2]
```diff
@@ -103,6 +103,17 @@
 	machine_understands = FALSE
 	flags = RESTRICTED | HIVEMIND //CHOMPstation Edit: Changining from WHITELISTED to RESTRICTED | Empathy should be shadekin exclusive. This probably breaks carbon shadekins ability to use emptathy, we dont use them and if it does it should be implemented like DIONA root talk. -shark
```
Comment (Kashargul) [inline code/modules/mob/language/station_vr.dm:105]: //CHOMPEdit Start

File: code/modules/mob/living/carbon/human/species/station/traits_vr/positive.dm [hunk 3]
```diff
@@ -35,17 +35,23 @@
...
 
 /datum/trait/positive/nonconductive
-	name = "Non-Conductive"
+	name = "Non-Conductive, Minor"
 	desc = "Decreases your susceptibility to electric shocks by 25%." //CHOMP Edit - GRAMMAR PLS.
 	cost = 2 //This effects tasers!
 	var_changes = list("siemens_coefficient" = 0.75) //CHOMP Edit
 
 /datum/trait/positive/nonconductive_plus
-	name = "Non-Conductive, Major"
```
Comment (Kashargul) [inline code/modules/mob/living/carbon/human/species/station/traits_vr/positive.dm:44]: // CHOMPEdit

File: code/modules/mob/living/carbon/human/species/station/traits_vr/positive.dm [hunk 4]
```diff
@@ -35,17 +35,23 @@
 	H.setMaxHealth(S.total_health)
 
 /datum/trait/positive/nonconductive
-	name = "Non-Conductive"
+	name = "Non-Conductive, Minor"
```
Comment (Kashargul) [inline code/modules/mob/living/carbon/human/species/station/traits_vr/positive.dm:38]: //CHOMPEdit

File: code/modules/mob/living/carbon/human/species/station/traits_vr/positive.dm [hunk 5]
```diff
@@ -35,17 +35,23 @@
...
 	desc = "Decreases your susceptibility to electric shocks by 25%." //CHOMP Edit - GRAMMAR PLS.
 	cost = 2 //This effects tasers!
 	var_changes = list("siemens_coefficient" = 0.75) //CHOMP Edit
 
 /datum/trait/positive/nonconductive_plus
-	name = "Non-Conductive, Major"
+	name = "Non-Conductive"
 	desc = "Decreases your susceptibility to electric shocks by 50%." //CHOMP Edit - GRAMMAR PLS.
 	cost = 3 //Let us not forget this effects tasers!
 	var_changes = list("siemens_coefficient" = 0.5) //CHOMP Edit
```
Comment (Kashargul) [inline code/modules/mob/living/carbon/human/species/station/traits_vr/positive.dm:48]: //CHOMPEdit Start

File: code/modules/mob/living/carbon/human/species/station/traits_vr/positive.dm [hunk 6]
```diff
@@ -35,17 +35,23 @@
...
 
+/datum/trait/positive/nonconductive_plus_plus
+	name = "Non-Conductive, Major"
+	desc = "Decreases your susceptibility to electric shocks by 100%." //CHOMP Edit
+	cost = 6
+	var_changes = list("siemens_coefficient" = 0) //CHOMP Edit
+
```
Comment (Kashargul) [inline code/modules/mob/living/carbon/human/species/station/traits_vr/positive.dm:54]: //CHOMPEdit End

File: code/modules/mob/living/carbon/human/species/station/traits_vr/positive_ch.dm [hunk 7]
```diff
@@ -407,3 +407,26 @@
 /datum/trait/positive/toxin_gut/apply(var/datum/species/S,var/mob/living/carbon/human/H)
 	..()
 	H.toxin_gut = TRUE
+
+/datum/trait/positive/nobreathe // CHOMPedit
```
Comment (Kashargul) [inline code/modules/mob/living/carbon/human/species/station/traits_vr/positive_ch.dm:411]: We are inside a .ch file, no // CHOMPedit comments here

**Comment by Cyrelius** [comment]
This trait, while expensive, is incredibly abusable, and due to making people immune to tasers, will cause more issues where security is involved.  In addition, it would make people immune to all other things that cause electric shocks, including tesla engines and lightning strikes.

**Comment by ItsSelis** [comment]
This PR is being rejected due to concerns that the trait may be abused or too overpowered. It would enable any individual, regardless of species, to be fully immune to electric shocks, including those from wires, Tesla and similar engines, electricity-based non-lethal weaponry, and lightning strikes.

**Pull request closed without merging**